db/data_migration/20160311093609_generate_content_ids_on_attachments.rb

```diff
@@ -0,0 +1,6 @@
+require 'securerandom'
+
+Attachment.where(content_id: nil).find_each do |attachment|
```

Comment:
Beware - using the MySQL UUID function risks having different UUIDs generated on the primary database compared to the replicas.  We've had this problem happen in the past.  We've deliberately generated UUIDs in Ruby to avoid this risk.